src/coordinator.coffee

```diff
@@ -244,6 +249,17 @@ class Coordinator extends EventEmitter
         delete @process[k]
       return callback null, processes
 
+  updateNodeMetadata: (node, metadata, callback) ->
+    metadata = {} unless metadata
+    process = null
+    for k, v of @participants
```

Comment:
I don't get it. `startParticipant` adds metadata to `@processes`, but this updates metadata on `@participants` (but only the last one that matches)?
I don't see how these two data sources are reconsiled...